src/functions/proximal_maps.jl

```diff
@@ -316,7 +316,7 @@ function prox_TV2(
             "Proximal Map of TV2(M, λ, x, p) not implemented for p=$(p) (requires p=1) on general manifolds.",
         ),
     )
-    PowX = SVector(x)
+    PowX = [x...]
     PowM = PowerManifold(M, NestedPowerRepresentation(), 3)
```

Comment:
Most likely `ProductManifold` would be a better fit but I didn't try yet.

Reply:
indeed for the fixed case of 3 product is maybe better.